src/components/network/NetworkItem.vue

```diff
@@ -14,6 +14,41 @@
         {{ networkitem.chain_id }}
       </p>
     </div>
+    <div v-if="networkitem.powered" class="powered-div">
```

Comment:
since we're duplicating this HTML and associated styles this should really be its own component.